efficientdet/keras/efficientdet_keras.py

```diff
@@ -393,7 +394,9 @@ def __init__(self,
 
       bn_per_level = {}
       for level in range(self.min_level, self.max_level + 1):
-        bn_per_level[level] = utils_keras.build_batch_norm(
+        # NOTE(tanmingxing): key must be str to support model save/load.
```

Comment:
Could we unified use array or dict?

Reply:
I played with dict/array, and decide to use arrays for its simplicity.